frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/emoji_icon_widget.dart

```diff
@@ -84,7 +84,7 @@ class RawEmojiIconWidget extends StatelessWidget {
           final iconData = IconsData.fromJson(jsonDecode(emoji.emoji));
           return IconWidget(
             data: iconData,
-            size: emojiSize,
+            size: emojiSize * 0.9,
```

Comment:
Why does the emoji size need to be multiplied by 0.9?

Reply:
It seems that for the same size, icons are larger than emojis

Before I changed the size:
![image](https://github.com/user-attachments/assets/ba13569b-44d9-4102-abef-f3d3c669c923)

After I changed the size:
<img width="132" alt="image" src="https://github.com/user-attachments/assets/245962c3-14a7-40d9-9a8b-f0d880122666" />

As for why it's 0.9, I referred to the code in flowy_infra_ui, perhaps I should add relevant comments
<img width="596" alt="image" src="https://github.com/user-attachments/assets/afd916b2-bd3a-42df-a26b-ad66837fc0fb" />

Reply:
<img width="155" alt="Screenshot 2024-12-30 at 10 17 12" src="https://github.com/user-attachments/assets/bff51982-6f72-4cfe-b696-1d1028a17729" />